pkgs/applications/misc/blender/default.nix

```diff
@@ -136,24 +129,28 @@ stdenv.mkDerivation rec {
       --add-flags '--python-use-system-env'
   '';
 
-  # Set RUNPATH so that libcuda and libnvrtc in /run/opengl-driver(-32)/lib
-  # can be found. See the explanation in libglvnd.
-  postFixup = lib.optionalString cudaSupport ''
-    for program in $out/bin/blender $out/bin/.blender-wrapped; do
-      isELF "$program" || continue
-      addOpenGLRunpath "$program"
-    done
-  '';
-
   meta = with lib; {
     description = "3D Creation/Animation/Publishing System";
     homepage = "https://www.blender.org";
     # They comment two licenses: GPLv2 and Blender License, but they say:
     # "We've decided to cancel the BL offering for an indefinite period."
-    # CUDA toolkit is unfree.
-    license = with licenses; [ gpl2Plus ]
-      ++ lib.optionals cudaSupport [ unfree ];
+    license = with licenses; [ gpl2Plus ];
     platforms = [ "x86_64-linux" "x86_64-darwin" ];
     maintainers = with maintainers; [ goibhniu veprbl ];
   };
+};
+in
+symlinkJoin {
+  name = "blender-${blender-unwrapped.version}";
+
+  paths = [ blender-unwrapped ];
```

Comment:
```
runCommandNoCC {
# ...

  unwrapped = blender-unwrapped;
  buildCommand = ''
    wrapProgram $unwrapped/foo/bar/blender
  '';
}
```
Allows to do things like
```nix
blender.overrideAttrs (_: { unwrapped = blender.unwrapped.overrideAttrs (_: { /* src = ...; */ }); })
```
As well as simply accessing the unwrapped derivation

See prior PR #97388